src/components/Card.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class Card extends React.Component {
```

Comment:
If you're only exporting one thing from a module you could do

`export default class Card extends React.Component {`.

Also if a component doesn't have state you can use PureComponents that are a bit more optimized.

`export default class Card extends React.PureComponent {`